site/quickstart.md

```diff
@@ -164,7 +172,7 @@ domain namespace (`sample-domain1-ns`) and the `domainHomeImageBase` (`oracle/we
 For example, assuming you named your copy `my-inputs.yaml`:
 ```
 $ cd kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image
-$ ./create-domain.sh -i my-inputs.yaml -o /some/output/directory -e -v
+$ ./create-domain.sh -i my-inputs.yaml -o /some/output/directory -e
```

Comment:
The domain home in image sample create-domain.sh requires -u <username> and -p <password> options.

Reply:
yes, it requires -u and -p options.

Eg: [root@llg00cpv domain-home-in-image]# ./create-domain.sh -i uidomain-create-domain-inputs.yaml -o /scratch/output/uidomain -e -u weblogic -p welcome1

Also regarding passing option "-e" to create-domain.sh, it's worth noting users to use that option only if it is a single node. If it is a multi-node cluster, the image would not be available across worker nodes.

Reply:
ok, will update

Reply:
Thanks, Mark. You will add the -u -p options in the create-domain command line?

Reply:
yeah i did, see latest update @rosemarymarano

Reply:
Thanks!!!